brother_ql/reader.py

```diff
@@ -113,7 +180,12 @@
   'Expansion area',
   'Tape color information',
   'Text color information',
-  'Hardware settings',
+  'Hardware settings 1',
+  'Hardware settings 2',
+  'Hardware settings 3',
+  'Hardware settings 4',
```

Comment:
Q (non-blocking): Is this the official name/variable per model or just unknown?

Reply:
It's a 4-byte field that isn't really documented anywhere. In this list one entry is one byte, so it's split up into 4 names.